trace/cloud-trace-demo-app/app/demo-service-c.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-trace-demo-c
+  labels:
+    app: cloud-trace-demo-app-c
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloud-trace-demo-app-c
+  template:
+    metadata:
+      name: cloud-trace-demo-c
+      labels:
+        app: cloud-trace-demo-app-c
+    spec:
+      containers:
+      - name: cloud-trace-demo-container
+        image: gcr.io/another-gke-demo/final-demo-image:latest
+        command:
+        - python
+        args:
+        - app.py
+        - --keyword=!
+        - --endpoint=http://34.83.190.145
```

Comment:
Should these endpoints be hardcoded?

Reply:
The file is generated from the template and is now removed.